.github/labeler.yml

```
@@ -16,3 +16,4 @@ github_actions:
   - changed-files:
     - any-glob-to-any-file:
       - '.github/workflows/*.yml'
```

yamllint (1.29.0-1)

[error] wrong indentation: expected 8 but found 6

(indentation)